lib/mayaUsd/resources/ae/usdschemabase/ae_template.py

```diff
@@ -660,6 +659,123 @@ def get(self):
             folderIndex[groups].items.append(attributeInfo.name)
         return self._attributeLayout
 
+class DisplayCustomDataControl(object):
+    '''Custom control for adding in extra display custom data.'''
+    def __init__(self, item, prim):
+        super(DisplayCustomDataControl, self).__init__()
+        self.item = item
+        self.prim = prim
+
+        # Check whether Ufe has the scene item meta data support.
+        self.useMetadata = hasattr(ufe.SceneItem, "getGroupMetadata") and hasattr(ufe.Value, "typeName")
```

Comment:
When we have Ufe.SceneItem metadata and Ufe.Value support use them, otherwise read/write directly to/from USD. This allows this feature to be supported in all versions of Maya.